test/actions/IncreaseLiquidity.spec.ts

```diff
@@ -199,25 +220,19 @@ describe('IncreaseLiquidity.sol', function () {
     tokenId = mintReceipt.events[mintReceipt.events.length - 1].args.tokenId;
   });
 
-  describe('doAction', function () {
+  describe('IncreaseLiquidity.increaseLiquidity()', function () {
     it('should correctly perform the add liquidity action', async function () {
       await PositionManager.connect(user).depositUniNft(await NonFungiblePositionManager.ownerOf(tokenId), [tokenId]);
 
       const poolTokenId = 1;
+      const liquidityPre = (await NonFungiblePositionManager.positions(poolTokenId)).liquidity;
```

Comment:
Let's use always the same const name for the initial liquidity across different test cases